pkg/tidb/conn.go

```diff
@@ -29,6 +29,11 @@ import (
 )
 
 func (f *Forwarder) GetDBConnProps() (host string, port int, err error) {
```

Comment:
we can change this function to `getDBConnProps`, and move the override from here to `dsnConfig.Addr = fmt.Sprintf("%s:%d", host, port)` in `OpenTiDB`. In this way you don't need to parse the override config, like: 

```go
if tidbEndpointOverride != nil {
  dsnConfig.Addr = *tidbEndpointOverride
}
```

Reply:
updated. PTAL